processing/api.py

```diff
@@ -73,4 +73,30 @@ def tag_recording(self, recording, label, metadata):
         )
         r.raise_for_status()
 
+    def delete_tracks(self, recording):
+        url = self.url + "/{}/tracks".format(recording["id"])
```

Comment:
FYI regarding a neat feature in Python 3.6+: f-strings. Example:

```python
url = self.url + f"/{recording["id"]}/tracks"
```

No need to change here - just passing on a useful feature that helps to make string formatting more readable.
	
cacophony-processing already relies on Python 3.6 so it's ok to rely on.

More here: https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python